ovn-tester/ovn_workload.py

```diff
@@ -780,6 +851,14 @@ def start(self):
         self.sbctl = ovn_utils.OvnSbctl(
             self.central_node, sb_conn, inactivity_probe
         )
+        # ovn-ic configuration
+        if self.cluster_cfg.n_az > 1:
+            self.icnbctl = ovn_utils.OvnIcNbctl(
+                None, f'tcp:{self.central_node.mgmt_ip}:6645', inactivity_probe
+            )
+            self.nbctl.set_global('ic-route-learn', 'true')
+            self.nbctl.set_global('ic-route-adv', 'true')
```

Comment:
To be inline with the comments I left on the ovn-fake-multinode we can remove the `n_az > 1`; IC will always be running, even if there's only a single AZ.

Reply:
@dceara ack, I will fix it